### DISCUSSION TITLE
OpenDTU in S-mieles eintragen ?

### DISCUSSION THREAD

**Discussion by GNCLORD-MDB** [tbnobody/OpenDTU#2424] [Q&A]
Ich habe ja einen WR mit der OpenDTU verbunden und alles läuft nun einwandfrei. 
ich habe nun noch ein weiteres BKW das nun eine WR mit Wlan hat. 
Diesen kann ich ja nicht in die openDTU eintragen. 
Weiß einer ob man die openDTU DTU in die S-miles APP als DTU eintragen kann, so das dort der WR aus der Open DTU angezeigt wird und ich so beide WR dann zusammen sehen und anzeigen kann?

**Comment by stefan123t** [comment]
Die OpenDTU will selbst nicht in die Hoymiles Cloud schreiben das war ja der Grundgedanke hinter der Entwicklung, die eigenen Daten nicht in einer fremden Cloud zu speichern. Mit OpenDTU kannst Du die Daten per MQTT in einer eigenen Datenbank speichern und auswerten.

Für die neueren WR mit eingebautem WLAN Modul und DTUBI gibt es eine Software von DennisOSRM die die Daten auch per MQTT zur Verfügung stellt.

Es gibt auch eine python Bibliothek von rovo89 die prinzipiell Daten vom PC mit python in die S-Miles Cloud schreiben kann. Diese simuliert praktisch die Funktionen einer echten Hoymiles DTU / DTUBI.
Der Zustand ist aber eher ein technisches Proof of Concept und noch nicht ausgearbeitet. Man müsste diese Funktion dann in OpenDTU nachbauen damit OpenDTU seine Daten auch in die S-Miles Cloud speichern könnte.
